Misc/NEWS.d/next/Library/2021-05-09-22-52-34.bpo-44089.IoANsN.rst

```diff
@@ -0,0 +1 @@
+Allow subclassing ``csv.Error`` in 3.10 (it was allowed in 3.9 and earlier).
```

Comment:
Maybe explain that it got broken in earlier version of 3.10; as it stands this reads like nothing has changed.